pgsync/sync.py

```diff
@@ -189,8 +189,8 @@ def setup(self):
                     user_defined_fkey_tables.setdefault(node.table, set([]))
                     user_defined_fkey_tables[node.table] |= set(columns)
             if tables:
-                self.create_triggers(schema, tables=tables)
                 self.create_view(schema, tables, user_defined_fkey_tables)
+                self.create_triggers(schema, tables=tables)
```

Comment:
was there a reason for this?

Reply:
i'm not sure and i do not familiar with python or pg .
the phenomenon is the  tables in schema.json, update will be affected by pgsync bootstrap step.

![image](https://user-images.githubusercontent.com/52872118/129296178-d004134b-3801-4577-9e35-60fc3d0d9443.png)